src/main/java/org/sonar/plugins/checkstyle/CheckstyleProfileImporter.java

```diff
@@ -61,6 +61,7 @@
         "SuppressionCommentFilter",
         "SuppressWithNearbyCommentFilter",
         "SuppressionXpathSingleFilter",
+        "SuppressionSingleFilter",
```

Comment:
Do we have to do anything special for a new filter?
I remember we have this weird dialog in sonar where we have to manually put in the filter's config.
Did you verify you can use this filter without issue?

Reply:
Guess I am reacting too fast, picture came up showing its usage.

Reply:
yes, it's working. i uploaded the images.
here again for this particular filter.

before filter:
![magic_number](https://user-images.githubusercontent.com/653739/65595721-a7368500-df95-11e9-9b67-981382161250.png)

filter:
![filter_conf](https://user-images.githubusercontent.com/653739/65595736-ac93cf80-df95-11e9-8fb6-0facb4bcf6e4.png)

after filter:
![suppression](https://user-images.githubusercontent.com/653739/65595755-b4537400-df95-11e9-85a0-15d0b6226b4f.png)
(line 5 got suppressed)